--- VTK-5.0.4/Utilities/DICOMParser/DICOMAppHelper.cxx.gcc43	2007-05-03 11:12:13.000000000 -0600
+++ VTK-5.0.4/Utilities/DICOMParser/DICOMAppHelper.cxx	2008-04-11 13:52:05.000000000 -0600
@@ -31,7 +31,7 @@
 
 #include <stdlib.h>
 #include <stdio.h>
-#include <string>
+#include <cstring>
 #include <math.h>
 #include <algorithm>
 #if defined(__BORLANDC__)
--- VTK/Utilities/DICOMParser/DICOMFile.cxx.gcc43	2005-07-26 14:02:22.000000000 -0600
+++ VTK/Utilities/DICOMParser/DICOMFile.cxx	2008-10-01 10:08:33.000000000 -0600
@@ -23,7 +23,7 @@
 #endif 
 
 #include <stdio.h>
-#include <string>
+#include <cstring>
 
 #include "DICOMConfig.h"
 #include "DICOMFile.h"
--- VTK/Utilities/DICOMParser/DICOMParser.cxx.gcc43	2006-03-24 08:59:50.000000000 -0700
+++ VTK/Utilities/DICOMParser/DICOMParser.cxx	2008-10-01 10:08:33.000000000 -0600
@@ -36,7 +36,7 @@
 #include <sys/types.h>
 #endif
 
-#include <string>
+#include <cstring>
 
 #include "DICOMConfig.h"
 #include "DICOMParser.h"
