diff -up VTK-6.1.0/CMake/vtkModuleMacros.cmake.tcllib VTK-6.1.0/CMake/vtkModuleMacros.cmake
diff -up VTK-6.1.0/Wrapping/Tcl/CMakeLists.txt.tcllib VTK-6.1.0/Wrapping/Tcl/CMakeLists.txt
--- VTK-6.1.0/Wrapping/Tcl/CMakeLists.txt.tcllib	2014-01-22 08:55:41.000000000 -0700
+++ VTK-6.1.0/Wrapping/Tcl/CMakeLists.txt	2015-02-03 16:12:42.635456315 -0700
@@ -261,7 +261,7 @@ endforeach()
 # Configure the Tcl package index file for the install tree.
 SET(VTK_TCL_SCRIPT_DIR "[file dirname [info script]]")
 IF(UNIX)
-  SET(VTK_TCL_LIBRARY_DIR "[file dirname [file dirname [file dirname [info script]]]]")
+  SET(VTK_TCL_LIBRARY_DIR "${CMAKE_INSTALL_PREFIX}/${VTK_INSTALL_LIBRARY_DIR}")
 ELSE(UNIX)
   SET(VTK_TCL_LIBRARY_DIR
       "[file join [file dirname [file dirname [file dirname [file dirname [info script]]]]] bin]")
diff -up VTK-6.1.0/Wrapping/Tcl/pkgIndex.tcl.in.tcllib VTK-6.1.0/Wrapping/Tcl/pkgIndex.tcl.in
--- VTK-6.1.0/Wrapping/Tcl/pkgIndex.tcl.in.tcllib	2014-01-22 08:55:41.000000000 -0700
+++ VTK-6.1.0/Wrapping/Tcl/pkgIndex.tcl.in	2015-02-03 16:12:42.635456315 -0700
@@ -7,7 +7,7 @@ package ifneeded vtkinit {@VTK_MAJOR_VER
       if {[catch "load {} $libName"]} {
         set libExt [info sharedlibextension]
         set currentDirectory [pwd]
-        set libFile [file join $libPath "$libPrefix$libName-@VTK_MAJOR_VERSION@.@VTK_MINOR_VERSION@$libExt"]
+        set libFile [file join $libPath "$libPrefix$libName$libExt"]
         if {[catch "cd {$libPath}; load {$libFile}" errorMessage]} {
           puts $errorMessage
         }
diff -up VTK-6.1.0/Wrapping/Tcl/vtkbase/vtkbase.tcl.in.tcllib VTK-6.1.0/Wrapping/Tcl/vtkbase/vtkbase.tcl.in
--- VTK-6.1.0/Wrapping/Tcl/vtkbase/vtkbase.tcl.in.tcllib	2015-02-03 16:12:42.814455894 -0700
+++ VTK-6.1.0/Wrapping/Tcl/vtkbase/vtkbase.tcl.in	2015-02-03 16:23:08.653411667 -0700
@@ -44,7 +44,7 @@ namespace eval ::vtk {
         }
 
         foreach dir $dirs {
-            set libname [file join $dir ${prefix}${name}-@VTK_MAJOR_VERSION@.@VTK_MINOR_VERSION@${ext}]
+            set libname [file join $dir ${prefix}${name}${ext}]
             if {[file exists $libname]} {
                 if {![catch {load $libname} errormsg]} {
                     # WARNING: it HAS to be "" so that pkg_mkIndex work (since
