diff -up VTK-6.1.0/CMake/vtkModuleTop.cmake.install-prefix VTK-6.1.0/CMake/vtkModuleTop.cmake
--- VTK-6.1.0/CMake/vtkModuleTop.cmake.install-prefix	2015-07-13 15:00:52.538379740 -0600
+++ VTK-6.1.0/CMake/vtkModuleTop.cmake	2015-07-13 15:03:30.197677726 -0600
@@ -304,14 +304,26 @@ configure_file(CMake/VTKConfig.cmake.in
 # Generate VTKConfig.cmake for the install tree.
 set(VTK_CONFIG_CODE "
 # Compute the installation prefix from this VTKConfig.cmake file location.
-get_filename_component(VTK_INSTALL_PREFIX \"\${CMAKE_CURRENT_LIST_FILE}\" PATH)")
+set(_vtk_installed_prefix \"${CMAKE_INSTALL_PREFIX}/${VTK_INSTALL_PACKAGE_DIR}\")
+set(_vtk_requested_prefix \"\${CMAKE_CURRENT_LIST_DIR}\")
+get_filename_component(_vtk_installed_prefix_full \"\${_vtk_installed_prefix}\" REALPATH)
+get_filename_component(_vtk_requested_prefix_full \"\${_vtk_requested_prefix}\" REALPATH)
+if (_vtk_installed_prefix_full STREQUAL _vtk_requested_prefix_full)
+  set(VTK_INSTALL_PREFIX \"${CMAKE_INSTALL_PREFIX}\")
+else ()
+  set(VTK_INSTALL_PREFIX \"\${CMAKE_CURRENT_LIST_DIR}\")")
+
 # Construct the proper number of get_filename_component(... PATH)
 # calls to compute the installation prefix.
 string(REGEX REPLACE "/" ";" _count "${VTK_INSTALL_PACKAGE_DIR}")
 foreach(p ${_count})
   set(VTK_CONFIG_CODE "${VTK_CONFIG_CODE}
-get_filename_component(VTK_INSTALL_PREFIX \"\${VTK_INSTALL_PREFIX}\" PATH)")
+  get_filename_component(VTK_INSTALL_PREFIX \"\${VTK_INSTALL_PREFIX}\" PATH)")
 endforeach(p)
+
+set(VTK_CONFIG_CODE "${VTK_CONFIG_CODE}
+endif ()")
+
 set(VTK_CONFIG_CODE "${VTK_CONFIG_CODE}
 set(VTK_MODULES_DIR \"\${VTK_INSTALL_PREFIX}/${VTK_INSTALL_PACKAGE_DIR}/Modules\")")
 set(VTK_CONFIG_CMAKE_DIR "\${VTK_INSTALL_PREFIX}/${VTK_INSTALL_PACKAGE_DIR}")
