diff -up VTK-7.1.1/Utilities/KWIML/vtkkwiml/test/CMakeLists.txt.format VTK-7.1.1/Utilities/KWIML/vtkkwiml/test/CMakeLists.txt
--- VTK-7.1.1/Utilities/KWIML/vtkkwiml/test/CMakeLists.txt.format	2017-03-20 09:26:17.000000000 -0600
+++ VTK-7.1.1/Utilities/KWIML/vtkkwiml/test/CMakeLists.txt	2017-05-07 14:10:46.231254800 -0600
@@ -10,7 +10,7 @@ endif()
 # Suppress printf/scanf format warnings; we test if the sizes match.
 foreach(lang C CXX)
   if(KWIML_LANGUAGE_${lang} AND CMAKE_${lang}_COMPILER_ID STREQUAL "GNU")
-    set(CMAKE_${lang}_FLAGS "${CMAKE_${lang}_FLAGS} -Wno-format")
+    set(CMAKE_${lang}_FLAGS "${CMAKE_${lang}_FLAGS} -Wno-format -Wno-format-security")
   endif()
 endforeach()
 
