diff -ru VTK-9.1.0/ThirdParty/kissfft/vtkkissfft/CMakeLists.txt VTK-9.2.0.rc1/ThirdParty/kissfft/vtkkissfft/CMakeLists.txt
--- VTK-9.1.0/ThirdParty/kissfft/vtkkissfft/CMakeLists.txt      2021-11-04 13:48:20.000000000 -0600
+++ VTK-9.2.0.rc1/ThirdParty/kissfft/vtkkissfft/CMakeLists.txt  2022-06-20 10:08:06.000000000 -0600
@@ -40,3 +40,7 @@
 target_include_directories(kissfft
   PRIVATE
     "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>")
+
+if (UNIX)
+  vtk_module_link(VTK::kissfft PRIVATE m)
+endif ()
