diff -up VTK-9.0.1/Rendering/FreeTypeFontConfig/vtkFontConfigFreeTypeTools.cxx.freetype VTK-9.0.1/Rendering/FreeTypeFontConfig/vtkFontConfigFreeTypeTools.cxx
--- VTK-9.0.1/Rendering/FreeTypeFontConfig/vtkFontConfigFreeTypeTools.cxx.freetype	2021-01-23 17:24:14.600721579 -0700
+++ VTK-9.0.1/Rendering/FreeTypeFontConfig/vtkFontConfigFreeTypeTools.cxx	2021-01-23 17:25:05.457127603 -0700
@@ -29,6 +29,13 @@ vtkStandardNewMacro(vtkFontConfigFreeTyp
 namespace
 {
 // The FreeType face requester callback:
+#ifndef FT_CALLBACK_DEF
+#ifdef __cplusplus
+#define FT_CALLBACK_DEF( x )  extern "C"  x
+#else
+#define FT_CALLBACK_DEF( x )  static  x
+#endif
+#endif /* FT_CALLBACK_DEF */
 FT_CALLBACK_DEF(FT_Error)
 vtkFontConfigFreeTypeToolsFaceRequester(
   FTC_FaceID face_id, FT_Library lib, FT_Pointer request_data, FT_Face* face)
diff -up VTK-9.0.1/Rendering/FreeType/vtkFreeTypeTools.cxx.freetype VTK-9.0.1/Rendering/FreeType/vtkFreeTypeTools.cxx
--- VTK-9.0.1/Rendering/FreeType/vtkFreeTypeTools.cxx.freetype	2021-01-23 17:24:14.600721579 -0700
+++ VTK-9.0.1/Rendering/FreeType/vtkFreeTypeTools.cxx	2021-01-23 17:25:49.546479602 -0700
@@ -378,6 +378,13 @@ FTC_CMapCache* vtkFreeTypeTools::GetCMap
 }
 
 //----------------------------------------------------------------------------
+#ifndef FT_CALLBACK_DEF
+#ifdef __cplusplus
+#define FT_CALLBACK_DEF( x )  extern "C"  x
+#else
+#define FT_CALLBACK_DEF( x )  static  x
+#endif
+#endif /* FT_CALLBACK_DEF */
 FT_CALLBACK_DEF(FT_Error)
 vtkFreeTypeToolsFaceRequester(
   FTC_FaceID face_id, FT_Library lib, FT_Pointer request_data, FT_Face* face)
